python/repositories.bzl

```diff
@@ -349,6 +395,7 @@ def python_register_toolchains(
         distutils: see the distutils attribute in the python_repository repository rule.
         distutils_content: see the distutils_content attribute in the python_repository repository rule.
         register_toolchains: Whether or not to register the downloaded toolchains.
+        set_python_version_constraint: When set to true, target_compatible_with for the toolchains will include a version constraint.
```

Comment:
Why does this need to be an opt-in setting instead of on by default?

Reply:
This allows a default toolchain to be picked when multi-python support is not enabled.